api/src/main/resources/underlays/synpuf.yaml

```diff
@@ -395,3 +406,16 @@ relationshipMappings:
         dataset: omop_test
         table: condition_occurrence
         column: condition_concept_id
+hierarchies:
```

Comment:
Not really related to this PR, but to capture it somewhere: from the discussion on Wednesday, it seems likely we will need to support multiple hierarchies eventually if we want feature parity for AoU. Something to keep in mind for the design, definitely not needed for the initial milestone.